### DISCUSSION TITLE
Case sensitive /app/pages

### DISCUSSION THREAD

**Discussion by SigurdMW** [blitz-js/blitz#1006] [General] [answered]
Hi, I added a page in `app/pages` called `About.tsx`. This gives a 404 error for this page. However, when renaming the file to `about.tsx`, it works. Is this how we want it to work? In my opinion, it should not matter.

**Comment by flybayer** [comment, marked as answer]
Hey Sigurd, If you name the page `About.tsx`, then the route will be `/About`. Check out [this discussion](https://github.com/vercel/next.js/discussions/15539) for a bit more context.

So if we made it all lowercase, then you wouldn't be able to have uppercase urls if you wanted.

**Reply by SigurdMW** [reply]
Thanks man, makes sense. Turns out Chrome (at least in my case) rewrites the URL from /About to /about and that's why I got 404 on the route. Probably because I have visited the lowercase version before
